controllers/user.js

```diff
@@ -230,6 +232,20 @@ async function getDashboard(req, res) {
   const allBreaches = req.app.locals.breaches;
   const { verifiedEmails, unverifiedEmails } = await getAllEmailsAndBreaches(user, allBreaches);
 
+  let experimentBranch = null;
+  let isUserInExperiment = null;
+  let experimentBranchB = null;
+
+  if (EXPERIMENTS_ENABLED) {
+    const coinFlipNumber = Math.floor(Math.random() * 100);
+    experimentBranch = getExperimentBranch(req, coinFlipNumber);
```

Comment:
This looks like if someone never does a scan, but gets straight to their dashboard, they'll be put into an experiment?